test/util.js

```diff
@@ -24,14 +24,17 @@ async function testServer (handler, sessionOptions, plugin) {
 async function request (options) {
   let response
   try {
-    response = await got(options)
+    if (typeof options === 'string') {
+      response = await undici.request(options)
+    } else {
+      response = await undici.request(options.url, options)
+    }
   } catch (err) {
     response = err.response
   }
   const { statusCode, body } = response
-  const cookieHeader = response.headers['set-cookie']
-  const cookie = cookieHeader ? cookieHeader[0] : null
-  return { response, body, statusCode, cookie }
+  const cookie = response.headers['set-cookie']
+  return { response, body: await body.text(), statusCode, cookie }
```

Comment:
I already gave approval, but you should consider what happens if `body.text()` fails. With `got`, the error would have already been caught in the above `try/catch`.